tools/apidoc/gen_toc.py

```diff
@@ -171,6 +171,9 @@
     'StratosphereSsp' : ' Stratosphere SSP',
     'Metrics' : 'Metrics',
     'Infrastructure' : 'Metrics',
+    'listAnnotations' : 'Annotations',
```

Comment:
We can re-write this as:
```
'Annotations' : 'Annotations'
```
This will pick and group any apis under 'Annotations' that have the substring